flipperdevices/flipperzero-firmware
# [FL-1934] Core: wipe memory after free. SubGhz: key encryption tool.

# What's new

- Core: wipe memory after free.
- RFID, iButton: fix iterator use after invalidation.
- SubGhz: new `subghz` tool
- SubGhz: support encrypted keeloq_mfkeys
- FuriHalCrypto: fix IV corruption for CBC
- Makefile: move formatting to top Makefile, conditional flashing and debug
- Debug: recursive register print in svd command, update and minify svd file

# Verification 

- Compile and upload
- Check subghz  tool
- Deploy keeloq_mfcodes and verify that decryption works

# Checklist (do not modify)
